admin_guide/revhistory_admin_guide.adoc

```diff
@@ -41,6 +24,14 @@ n|xref:../admin_guide/scheduling/scheduler.adoc#admin-guide-scheduler[Default Sc
 |xref:../admin_guide/service_accounts.adoc#admin-guide-service-accounts[Configuring Service Accounts]
 |Noted the importance of `-z` flag usage when xref:../admin_guide/service_accounts.adoc#admin-sa-user-names-and-groups[granting access to service accounts].
 
+|xref:../admin_guide/scheduling/scheduler.adoc#admin-guide-scheduler[Default Scheduling]
```

Comment:
Note that I reorganized the revision history since this is for 3.9 with a release date of March 21 and all changes should fall under that date for now.